docs/ILLUSTRATOR_GUIDE.md

```diff
@@ -0,0 +1,21 @@
+# Illustrator Template Guide
+
+This Guide explains how to properly use the Adobe Illustrator Template for Lucide. 
+>Attention: Even though it is unlikely the template can be outdated or not 100% correct. Please check the Icon Design Guide before you start working with the template to ensure integrity with the Lucide icon pack.
+  
+## General Workflow
+
+The Illustrator template is created following guidelines from the [Icon Design Guide](ICON_DESIGN_GUIDE.md). 
+  
+**Workflow:**
+1. Open the Document which can be found under __*/docs/templates/illustrator_template.ai*__ .
+
+2. You can now remove the content from the example logo layer ("Draw") and start creating.
+
+3. Verify that you follow the [Icon Design Guidelines](ICON_DESIGN_GUIDE.md).
+
+4. When you are ready to export the file as an SVG make sure to check that you followed the guidelines and remove all unecessary layers (especially "Padding" and "Grid").
+
+5. Double check that the [code conventions and SVG global attributes](https://github.com/lucide-icons/lucide/blob/master/docs/ICON_DESIGN_GUIDE.md#code-conventions) are correct.
+
+6. Minify paths with [SVGOMG](https://jakearchibald.github.io/svgomg/).
```

Comment:
Maybe nice to write also something about exporting designs to SVG.
Illustrator has an excellent exporting tool, called "Export as" in the file menu.
Here is an article about it: [Export as article - CSS Tricks](https://css-tricks.com/illustrator-to-svg/#the-export-as-method)

Reply:
Does the minify option in the export menu do the same thing as SVGOMG? Would be important to know as that could avoid redundancies in the workflow.

What would you recommend as standard export settings?
I would say: 

- **Style**: `internal CSS` (easier to overwrite and remove)
- **Font**: probably doesn't matter but I would go with `Convert to Outlines` (font independent but larger)
- **Images**: don't know if we need to change that
- **Object-IDs**: I would go with `unique` to avoid conflicts
- **Decimal**: `2` is fine
- **minify**: `yes` cause why not. Seems to work properly
- **responsive**: `no` because width and height are required attributes according to the [IDG - Global Attributes](https://github.com/lucide-icons/lucide/blob/master/docs/ICON_DESIGN_GUIDE.md#code-conventions)

Reply:
Yeah, SVGOMG is the best, but illustrator can compress a lot as well. I would say do both

The preferences matching the best with lucides codestyle are: 
- **Style**: `Presentation Attributes`
- **Font**:  `Convert to Outlines`, 
- **Images**: no config, we don't accept images
- **Object-IDs**: `none` (if possible)
- **Decimal**: `2` or less depending if curves are used.
- **minify**: `yes` cause why not. Seems to work properly
- **responsive**: `no`

Reply:
![image](https://user-images.githubusercontent.com/72730682/129181991-3291291b-fd26-4a4d-a4d2-c498412a9c35.png)
so those are the best settings.